src/layer/layer_settings_util.cpp

```diff
@@ -230,34 +230,34 @@ std::vector<VlFrameset> ToFrameSets(const std::string &s) {
 }
 
 bool IsFrameSets(const std::string &s) {
-    static const std::regex FRAME_REGEX("^([0-9]+([-][0-9]+){0,2})(,([0-9]+([-][0-9]+){0,2}))*$");
+    static const std::regex frame_regex("^([0-9]+([-][0-9]+){0,2})(,([0-9]+([-][0-9]+){0,2}))*$");
 
-    return std::regex_search(s, FRAME_REGEX);
+    return std::regex_search(s, frame_regex);
 }
 
 bool IsInteger(const std::string &s) {
-    static const std::regex FRAME_REGEX("^-?([0-9]*|0x[0-9|a-z|A-Z]*)$");
+    static const std::regex frame_regex("^-?([0-9]*|0x[0-9|a-z|A-Z]*)$");
 
-    return std::regex_search(s, FRAME_REGEX);
+    return std::regex_search(s, frame_regex);
 }
 
 bool IsFloat(const std::string &s) {
-    static const std::regex FRAME_REGEX("^-?[0-9]*([.][0-9]*f?)?$");
+    static const std::regex frame_regex("^-?[0-9]*([.][0-9]*f?)?$");
 
-    return std::regex_search(s, FRAME_REGEX);
+    return std::regex_search(s, frame_regex);
 }
 
 std::string FormatString(const char *message, ...) {
-    std::size_t const STRING_BUFFER(4096);
+    std::size_t const string_buffer(4096);
```

Comment:
string_buffer is a confusing name since its the *length* of the buffer, not the buffer itself.